ui/widgets/nuxeo-tag-suggestion.js

```diff
@@ -243,10 +243,23 @@ import { escapeHTML } from './nuxeo-selectivity.js';
     _addedTagHandler(entry) {
       if (this.document) {
         this.$.addTagOp.params = { tags: entry.id };
-        this.$.addTagOp.execute().then(() => {
+        this.$.addTagOp.execute().then((response) => {
+          const lenArr = response.properties['nxtag:tags'].length;
```

Comment:
add null checks on response and response.properties['nxtag:tags'] using optional chaining operator